taskcluster/fenix_taskgraph/transforms/beetmover.py

```diff
@@ -50,14 +50,20 @@ def make_task_description(config, tasks):
         if task.get("locale"):
             attributes["locale"] = task["locale"]
 
+        # Switch between nightly and release scope
+        # Note: if dep bucket is wanted, *temporarily* switch it to "~:dep"
+        if task['build-type']['nightly']:
+            bucket_scope = "project:mobile:fenix:releng:beetmover:bucket:nightly"
+        else:
+            bucket_scope = "project:mobile:fenix:releng:beetmover:bucket:release"
```

Comment:
Should we check for beta or release here, and default to dep? Or check level and send `level < 3` to dep?

Reply:
This is a candidate for a `by-level / by-build-type` nested config :)

Reply:
we return straight json here, where would `by-*` keys be calculated?

Reply:
We could have the `by-` config in the kind.yml under some extra key, and here in the transform we can pop that key, evaluate the keyed_by, and set the scopes accordingly. Does that make sense?

Reply:
I was thinking more:

[here](https://github.com/mozilla-mobile/fenix/blob/8de4c0b4dbcc185c522903f321030770591c0198/taskcluster/ci/beetmover/kind.yml#L24):
```
job-template:
    scopes:
        by-level:
            '3':
                 by-build-type:
                     'nightly':  # nightly
                     (release|beta):  # release
                      default:  # dep?
            default:  # use dep
```

Thoughts?